go/vt/wrangler/tablet.go

```diff
@@ -201,3 +208,23 @@ func (wr *Wrangler) VReplicationExec(ctx context.Context, tabletAlias *topodatap
 	}
 	return wr.tmc.VReplicationExec(ctx, ti.Tablet, query)
 }
+
+func (wr *Wrangler) isMasterTablet(ctx context.Context, ti *topo.TabletInfo) (bool, error) {
```

Comment:
We should document in the function comment that even this is not the true way to determine whether a tablet is master - it's a shortcut. The true way would be to list all tablets in the shard in all cells and see which one has the highest timestamp among all those that claim to be master.